### ISSUE TITLE
login and logout are verbs, which should not appear in RESTful urls.

### ISSUE THREAD

**Issue by modricwang** [Midor1/ExpertHubAuth#1]
Consider using session?
POST session means one session is created, so an user logins.
so as DELETE session.

**Comment by modricwang** [comment]
modify issue title.

**Comment by Midor1** [comment]
I totally agree with you that verbs should not appear in RESTful APIs. I personally considered using HTTP method DELETE and PUT, too.
However, in some browsers, for these complicated methods, they will send an OPTION request first to acquire available methods from the back-end, which adds unnecessary complexity to the project.
After consideration, I decided to use verbs to avoid these HTTP methods and make the API more simple, although it looks not that much "REST".
